cosmo_tester/framework/util.py

```diff
@@ -149,13 +149,24 @@ def create_rest_client(
 
 def get_plugin_wagon_urls():
     """Get plugin wagon urls from the cloudify-versions repository."""
-    plugin_urls_location = (
-        'https://raw.githubusercontent.com/cloudify-cosmo/'
-        'cloudify-versions/{branch}/packages-urls/plugin-urls.yaml'.format(
-                branch=os.environ.get('BRANCH_NAME_CORE', 'master'),
-        )
-    )
-    return yaml.load(requests.get(plugin_urls_location).text)['plugins']
+    branch = os.environ.get('BRANCH_NAME_CORE', 'master')
+    url_format = 'https://raw.githubusercontent.com/cloudify-cosmo/' \
+        'cloudify-versions/{branch}/packages-urls/plugin-urls.yaml'
+    plugin_urls_location = (url_format.format(branch=branch))
+    response = requests.get(plugin_urls_location)
+    if response.status_code != 200:
+        if branch != 'master':
+            plugin_urls_location = (url_format.format(branch='master'))
+            response = requests.get(plugin_urls_location)
+            if response.status_code != 200:
+                raise RuntimeError(
+                    'Response to HEAD request was {status}'.format(
+                        status=response.status_code))
```

Comment:
None clear error message, maybe something like "can not access.."

Reply:
Should be clearer now.